src/pages/workspace/tags/EditTagPage.tsx

```diff
@@ -39,22 +38,15 @@ function EditTagPage({route, policyTags}: EditTagPageProps) {
     const {inputCallbackRef} = useAutoFocusInput();
     const currentTagName = route.params.tagName;
 
-    const validate = useCallback(
-        (values: FormOnyxValues<typeof ONYXKEYS.FORMS.WORKSPACE_TAG_FORM>) => {
-            const errors: FormInputErrors<typeof ONYXKEYS.FORMS.WORKSPACE_TAG_FORM> = {};
-            const tagName = values.tagName.trim();
-            const {tags} = PolicyUtils.getTagList(policyTags, 0);
-
-            if (!ValidationUtils.isRequiredFulfilled(tagName)) {
-                errors.tagName = 'workspace.tags.tagRequiredError';
-            } else if (tags?.[tagName]) {
```

Comment:
@waterim Why do you remove this validation? We still need to check whether the new name is duplicated with name of other tags

Reply:
Yes, agree